### PR TITLE
Add bombastic walker

### PR DESCRIPTION
**Pull request by gildub**
After running `journalctl -xeu bombastic-walker.service`, run `systemctl status bombastic-api.service` to verify the api is accepting SBOMs : 

image

### CODE REVIEW

File: roles/tpa_single_node/templates/manifests/bombastic/walker/Deployment.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        - image: "{{ tpa_single_node_trustification_image }}"
+          imagePullPolicy: Always
+          name: walker
+          command:
+            - /trust
+          args:
+            - bombastic
+            - walker
+            - --sink
+            - http://127.0.0.1:{{ tpa_single_node_bombastic_api_port }}/api/v1/sbom
```
Comment (mrrajan) [inline roles/tpa_single_node/templates/manifests/bombastic/walker/Deployment.yaml.j2]: I hope it should be http://bombastic-api-pod:{{ tpa_single_node_bombastic_api_port }}/api/v1/sbom as the walker needs to reach out bombastic api pod and the localhost represent the bombastic walker pod.

Comment (gildub) [reply roles/tpa_single_node/templates/manifests/bombastic/walker/Deployment.yaml.j2]: @mrrajan, you're right, podman behavior is different than k8s cluster, thanks for catching that !

File: roles/tpa_single_node/templates/manifests/bombastic/indexer/Deployment.yaml.j2 [hunk 2]
```diff
@@ -1,3 +1,4 @@
+apiVersion: apps/v1
```
Comment (desmax74) [inline roles/tpa_single_node/templates/manifests/bombastic/indexer/Deployment.yaml.j2:1]: Is it needed with Podman kube play ?

File: roles/tpa_single_node/templates/manifests/bombastic/walker/Deployment.yaml.j2 [hunk 3]
```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
```
Comment (desmax74) [inline roles/tpa_single_node/templates/manifests/bombastic/walker/Deployment.yaml.j2:1]: Is it needed with Podman kube play ?

File: roles/tpa_single_node/templates/manifests/bombastic/indexer/Deployment.yaml.j2 (hunk 2 above)
Comment (gildub) [reply roles/tpa_single_node/templates/manifests/bombastic/indexer/Deployment.yaml.j2:1]: Not 100% sure it makes a difference, I thought it does, but we need to be consistent, so let's keep it for now and double check to eventually remove it later.

**Review by desmax74 — APPROVED** [review]

**Pull request merged**